src/vmware/azext_vmware/aaz/latest/vmware/private_cloud/_wait.py

```diff
@@ -198,7 +198,9 @@ def _build_schema_on_200(cls):
                 serialized_name="dnsZoneType",
             )
             properties.encryption = AAZObjectType()
-            properties.endpoints = AAZObjectType()
+            properties.endpoints = AAZObjectType(
+                flags={"read_only": True},
+            )
```

Comment:
I see that many properties have been marked as read-only. May I ask what is the reason?

Reply:
It's not due to swagger change as swagger has had these values since it was created. 

I believe that it's due to the python version upgrade and aaz-dev-tools upgrades. I have double checked and the properties do exist in swagger, so there could have been upgrades to code-gen tool that now capture these property flags.